docs/eventing/samples/writing-receive-adapter-source/05-receive-adapter.md

```diff
@@ -35,80 +36,34 @@ type Adapter struct {
 	logger   *zap.Logger
 	interval time.Duration
 	nextID   int
-	sink     cloudevents.Client
+	client   cloudevents.Client
 }
 ```
 
-The second required function is the `Start` function implmented as an interface to the adapter struct.
-for example:
+2. `Start` function implemented as an interface to the adapter struct.
 ```go
 func (a *Adapter) Start(stopCh <-chan struct{}) error {
 ```
-Where `stopCh` is the signal to stop the Adapter.  Otherwise the role of the funtion is to process the next
-event.  In the case of the `sample-source`, it creates an event to forward to the specificed cloudevent sink/client
-every X interval, as specified by the loaded EnvConfigAccessor (loaded via the resource yaml).
+`stopCh` is the signal to stop the Adapter.  Otherwise the role of the function is to process the next
+event.  In the case of the `sample-source`, it creates an event to forward to the specified cloudevent sink/client
+every X interval, as specified by the loaded `EnvConfigAccessor` (loaded via the resource yaml).
 ```go
 func (a *Adapter) Start(stopCh <-chan struct{}) error {
-	a.logger.Info("Starting with: ",
-		zap.String("Interval: ", a.interval.String()))
+    a.logger.Infow("Starting heartbeat", zap.String("interval", a.interval.String()))
 	for {
 		select {
 		case <-time.After(a.interval):
 			event := a.newEvent()
-			a.logger.Info("Sending new event: ", zap.String("event", event.String()))
-			_, _, err := a.sink.Send(context.Background(), event)
-			if err != nil {
-				return err
-			}
+			a.logger.Infow("Sending new event", zap.String("event", event.String()))
+			if result := a.client.Send(context.Background(), event); !cloudevents.IsACK(result) {
+                a.logger.Infow("failed to send event", zap.String("event", event.String()), zap.Error(result))
+                // We got an error but it could be transient, try again next interval.
+                continue
+            }
 		case <-stopCh:
 			a.logger.Info("Shutting down...")
 			return nil
 		}
 	}
 }
 ```
-
-## Reconcile/Create The Receive Adapter
-As part of the source reconciliation, we have to create and deploy
-(and update if necessary) the underlying receive adapter.  The two
-client sets used in this process is the `kubeClientSet` for the
-Deployment tracking, and the `EventingClientSet` for the event
-recording.
-
-Verify the specified kubernetes resources are valid, and update the `Status` accordingly
-
-Assemble the ReceiveAdapterArgs
```

Comment:
This section was actually duplicate of https://github.com/knative/docs/pull/2475/files#diff-df6286b13f3371b30a72bf3277b63a96R30